package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "grunt-dot-compiler",
   "description": "Grunt task for compiling doT templates",
-  "version": "0.5.12",
+  "version": "1.0.0",
```

Comment:
I think it would be better to keep 0.5.12 as a reference, maybe we can add an extra number at the end like 0.5.121

Reply:
Cool